### ISSUE TITLE
F-Droid build failed

### ISSUE THREAD

**Issue by linsui** [Blockstream/green_android#140]
### Description

```
build.sh: unrecognized option '--enable-rust'
```

This option should be removed.

**Comment by angelix** [comment]
Fixed in 0d9e341d2ef7ad667b64ade408a11d731e6f4336
Thanks @linsui for reporting.

**Comment by linsui** [comment]
I got this error. Could please take a look?

```
   Compiling gdk-rust v0.1.0 (/builds/linsui/fdroiddata/build/com.greenaddress.greenbits_android_wallet/crypto/gdk/build-clang-android-armeabi-v7a/subprojects/gdk_rust/gdk_rust)
    Finished release [optimized] target(s) in 6m 03s
/opt/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-objcopy: error: '/builds/linsui/fdroiddata/build/com.greenaddress.greenbits_android_wallet/crypto/gdk/build-clang-android-armeabi-v7a/subprojects/gdk_rust/libgdk_rust.a(bio_ssl.o)': The file was not recognized as a valid object file
ninja: build stopped: subcommand failed.
```

Thanks!
